packages/paste-website/src/components/DoDont.tsx

```diff
@@ -4,78 +4,68 @@ import {themeGet} from 'styled-system';
 import {Box} from '@twilio-paste/box';
 import {Text} from '@twilio-paste/text';
 
+interface DoDontProps {
+  children: React.ReactNode;
+}
+
 const StyledWrapper = styled(Box)`
   display: grid;
   grid-template-columns: repeat(2, 1fr);
   grid-gap: ${themeGet('space.space70')};
 `;
 
-interface DoDontProps {
-  children: React.ReactNode;
-}
-
 const DoDont: React.FC<DoDontProps> = props => {
   return (
-    <StyledWrapper marginTop="space90" marginBottom="space90">
+    <StyledWrapper marginTop="space90" marginBottom="space130">
       {props.children}
     </StyledWrapper>
   );
 };
 
-interface ExampleProps {
-  children: React.ReactNode;
-  do: boolean;
-  image?: string;
-}
-
-const StyledExampleWraper: React.FC<ExampleProps> = styled(Box)`
-  border-top: ${(props: ExampleProps) =>
-    props.do
-      ? `${themeGet('borderWidths.borderWidth20')(props)} solid #23bf6e`
-      : `${themeGet('borderWidths.borderWidth20')(props)} solid #ce241a`};
+const DoDontImage = styled(Box)`
+  border: ${themeGet('borderWidths.borderWidth10')} solid #ccd2dc;
+  border-bottom: 0;
 `;
 
-const StyledExampleImg: React.FC<ExampleProps> = styled(Box)`
+const DoDontComponent = styled(Box)`
```

Comment:
Can use the `Absolute` component here also.

Reply:
Good catch. I'll make that change.